golden_test/README.md

```diff
@@ -0,0 +1,22 @@
+# Golden tests
+Golden tests, also called screenshot tests, let's you write tests to easily find UI regressions. It
+compares the UI to saved goldens screenshots, and if there is a pixel difference the tests will
+fail. It gives us confidence that code changes does not cause UI regressions.
+
+## Updating goldens
+An expected change to the UI will also cause the golden tests to fail. You will then need to update
+the goldens by running the following command on your macOS machine:
+```bash
+flutter test --update-goldens golden_test
+```
+
+Due to slight [differences](https://github.com/flutter/flutter/issues/36667#issuecomment-521335243)
+in rendering across platforms, mostly around text, the tests will only be run on a macOS machine
+on Github Actions. This means that if you update the tests on a Linux or Windows machine the golden
+tests will not pass on Github Actions. Instead you are recommended to download the goldens directly
+from the failed Github Actions job, and use those inside of your branch.
+
+You can find the goldens under Artifacts in the top menu inside of the failed Github Actions job. It
+will include the master image, your test image and the difference.
+
+![Where to download the golden artifacts](https://user-images.githubusercontent.com/1770678/80202787-70f1de80-8626-11ea-9d98-58ac72f67479.png)
```

Comment:
Add a section explaining why we have to do font loading.

Reply:
Done.